### PR TITLE
Fixes brass box hinges

### PR DESCRIPTION
**Pull request by ChangelingRain**
also they don't slow you down anymore because honestly that's just a worse dufflebag at that point, it's just a fancy box even if it is made of brass

### CODE REVIEW

File: code/game/objects/items/weapons/tools.dm [hunk 1]
```diff
@@ -644,6 +647,7 @@
 /obj/item/weapon/weldingtool/experimental/brass
 	name = "brass welding tool"
 	desc = "A brass welder that seems to constantly refuel itself. It is faintly warm to the touch."
+	resistance_flags = FIRE_PROOF | ACID_PROOF
```
Comment (coiax) [inline code/game/objects/items/weapons/tools.dm]: maybe declare the buffs to the tools

Comment (ChangelingRain) [inline code/game/objects/items/weapons/tools.dm]: c o n s i s t e n c y every magic brass thing is immune to both

Comment (coiax) [inline code/game/objects/items/weapons/tools.dm]: probably should list changes anyway though

File: code/game/objects/items/weapons/storage/toolbox.dm [hunk 2]
```diff
@@ -133,12 +133,13 @@
 	name = "brass box"
 	desc = "A huge brass box with several indentations in its surface."
```
Comment (Militaires) [inline code/game/objects/items/weapons/storage/toolbox.dm]: ON its surface

File: code/game/objects/items/weapons/storage/toolbox.dm [hunk 3]
```diff
@@ -133,12 +133,13 @@
 	name = "brass box"
 	desc = "A huge brass box with several indentations in its surface."
 	icon_state = "brassbox"
+	item_state = null
```
Comment (Militaires) [inline code/game/objects/items/weapons/storage/toolbox.dm]: oh hey it's just a guy running down the hall i shouldnt ready my weapon he's unarmed and shi OH GOD HE HAS AN INVISIBLE BOX OF MADE OUT OF FUCKING BRASS

File: code/game/objects/items/weapons/storage/toolbox.dm (hunk 2 above)
Comment (RemieRichards) [inline code/game/objects/items/weapons/storage/toolbox.dm]: it's an INdentation though.
There's a DENT - IN the object.

Indent:
"form deep recesses or notches in (a line or surface)." IN A SURFACE

File: code/game/objects/items/weapons/storage/toolbox.dm (hunk 3 above)
Comment (Haevacht) [inline code/game/objects/items/weapons/storage/toolbox.dm]: Aren't these boxes what cogscarabs get though, so it's obvious as is?

**Pull request merged**